mysql-test/suite/galera/t/pxc_install_gr.test

```diff
@@ -11,6 +11,8 @@ call mtr.add_suppression("Plugin group_replication reported");
 call mtr.add_suppression("Group replication cannot be used with Percona XtraDB Cluster in strict mode.");
 call mtr.add_suppression("Plugin 'group_replication' init function returned error.");
 call mtr.add_suppression("Using group replication with Percona XtraDB Cluster is only supported for migration");
+call mtr.add_suppression("Event [0-9]* Query apply failed: [0-9]*, seqno [0-9]*");
+call mtr.add_suppression("PLUGIN group_replication does not exist");
```

Comment:
Why do we need this suppression? Changing default value for wsrep_ignore_apply_errors should not trigger this warning. In current trunk works without this suppression.

Reply:
I assume you mean `call mtr.add_suppression("PLUGIN group_replication does not exist");` . That was indeed intriguing me. And it seems to be a matter of synchronization issue. Adding a simple sleep in between INSTALL and UNISTALL fixes the issue. I've raised a ticket for us to investigate it ( https://jira.percona.com/browse/PXC-3270 ) as it is not related to the default change (well spotted btw).